urdu/urdu-news.md

```diff
@@ -2,4 +2,6 @@ if you like to see urdu news [click here](http://www.bbc.com/urdu/)
 
 جن کو اردو آتی ہے آیے آپ کو اردو میں کہانی سناتے ہیں:
 
-خوشی کا عالم تھا کہ اچانک پلک جھپکتے ہی، نہ جانے کیوں، سب لوگوں کو سانپ سونگھ گیا-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.